app/views/payments/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<nav class="common-nav">
+  <%= link_to(image_tag("left-arrow.svg", alt: "Back Arrow"), "javascript:history.go(-1)", class: 'back-icon') %>
+</nav>
+
+<section class="new-expense-section">
+  <h2 class="form-title"> New Transaction </h2>
+
+  <%= form_with(model: [@expense, @payment], url: expense_payments_path(@expense), method: :post, id: 'payment-form', class: "field") do |form| %>
+    <div>
+      <%= form.label :name, class: "form-label" %>
+      <%= form.text_field :name, placeholder:"Name", class: "form-input" %>
+    </div>
+
+    <div>
+      <%= form.label :amount, class: "form-label" %>
+      <%= form.number_field :amount, placeholder:"Amount", class: "form-input" %>
+    </div>
+
+    <div class="checkbox-container">
+      <div class="checkboxes">
+        <div class: "form-label">Select Categories </div>
+        <% @expenses.each_with_index do |expense, index| %>
+          <%= check_box_tag 'payment[expense_ids][]', expense.id %>
+          <%= label_tag "payment_expense_#{expense.id}", expense.name %> <br>
```

Comment:
- [ ] I noticed that if I add a new expense by adding the `payment` detail, a user can select all categories at once and generate the payment. Which is not a good idea at all. Try to make your functionality in such a way that if a person clicks on one category then he/she should not be able to click on another category because it will make multiple payments across all categories.

![image](https://github.com/BHObserver/budget-app/assets/66476329/426758e9-9919-40b2-96b7-ed22099200dd)